packages/rich-text/src/test/remove-previous-line-separator.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+
+import { removePreviousLineSeparator } from '../remove-previous-line-separator';
+import { LINE_SEPARATOR } from '../special-characters';
+import { getSparseArrayLength } from './helpers';
+
+describe( 'removePreviousLineSeparator', () => {
```

Comment:
What's the expected behavior of an uncollapsed selection? I'd expect it to be different, at least based on the behavior of backspace in a traditional textarea field.

Reply:
I'll add tests for it as well. I guess it shouldn't remove the line.